### ISSUE TITLE
Find in article search box not visible in latest PWA code

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js-windows#732]
Something has broken the display of the find in article row of the navbar. It's still "working" (you can type a search term and press enter), but you can't see what you're typing or the options.

Could be a CSS issue, a z-index issue, or broken formatting in a recent update to index.html. Needs investigating and fixing.
